src/common/Navigation/styled.js

```diff
@@ -1,29 +1,125 @@
 import { NavLink } from "react-router-dom";
 import styled from "styled-components";
+import { ReactComponent as CameraIcon } from "../../images/camera.svg"
+import { ReactComponent as MagnifyingGlass } from "../../images/search.svg"
+import img from "../../images/search.svg";
 
 export const NavbarContainer = styled.nav`
-    height: 94px;
+    max-height: 94px;
     display: flex;
-    justify-content: center;
-    background-color: ${({theme}) => theme.color.black};
+    justify-content: space-evenly;
+    background-color: ${({ theme }) => theme.color.black};
+
+    @media(max-width: 860px) {
+        flex-wrap: wrap;
+        max-height: 144px;
+    }
+`;
+
+export const StyledNavbar = styled.ul`
+    list-style-type: none;
+    display: flex;
+    align-items: center;
+    padding-right: 7vw;
+
+    @media(max-width: 860px) {
+        padding: 0px;
+    }
 `;
 
 export const StyledNavLink = styled(NavLink)`
     padding: 8px 24px;
     gap: 10px;
     text-decoration: none;
-    color: ${({theme}) => theme.color.white};
+    color: ${({ theme }) => theme.color.white};
     font-size: 14px;
     line-height: 21px;
     &.active {
         border: 1px solid;
         border-radius: 24px;
     }
+
+    @media(max-width: 860px) {
+        display: flex;
+        align-items: center;
+        padding: 8px 1vw;
+        margin: 0px 1vw;
+        border-radius: 29px;
+}
 `;
 
-export const StyledNavbar = styled.ul`
-    list-style-type: none;
+export const StyledCameraIcon = styled(CameraIcon)`
+    g {
+        opacity: 1;
+    }
+
+    @media(max-width: 860px) {
+        width: 20px;
+        height: 21px;
```

Comment:
Took stroke-width from figma: 1.06
FIGMA:
<img width="404" alt="image" src="https://user-images.githubusercontent.com/54035117/175786966-32c3d139-7556-4c13-833a-f2de488a84ed.png">

PROJECT:
<img width="404" alt="image" src="https://user-images.githubusercontent.com/54035117/175786977-8ed54e56-64d3-44e1-a2eb-564700fccfec.png">


Font weight and size is the same as in figma and I'd argue if it looks better this way :/

Reply:
I also added the letter spacing -0.5px as in figma, now the letters are separated, but the camera icon is still suspiciously thin

Reply:
Hmm this is weird. I think this might have something to do with default svg viewbox of icon, but im not really sure what is at fault. Would having multiple svg files for different screen resolutions be ok? We need to adress this as soon as possible, because there are few places this icon is used for at least 2 screen resolutions.